### DISCUSSION TITLE
OpenAPI 3.1 support

### DISCUSSION THREAD

**Discussion by fegyi001** [directus/directus#21650] [Feature Requests]
### Summary

It would be nice if the generated OAS server specs would be in OpenAPI 3.1.x instead of 3.0.x.
Or maybe I'm missing something, but when I download the specs it is 3.0.1 by default and I cannot find any info whether a newer spec could be downloaded.

### Motivation

OpenAPI 3.1.x is out there for several years now (since 2021)

### Detailed Design

http://localhost:8055/server/specs/oas => this should output a json with `openapi: "3.1.0"`

### Must Have:
- generate json server documentation with openapi version 3.1.x

### Drawbacks

It should be backwards compatible.

### Alternatives

Strapi supports 3.1.x...

### Adoption Strategy

I don't know yet.

**Comment by rijkvanzanten** [comment]
What features of the OpenAPI spec are you missing that exist in 3.1 but not 3.0?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
